Lib/test/pythoninfo.py

```diff
@@ -155,7 +155,7 @@ def collect_platform(info_add):
 def collect_locale(info_add):
     import locale
 
-    info_add('locale.encoding', locale.getpreferredencoding(False))
+    info_add('locale.encoding', locale.getencoding())
```

Comment:
It might add confusion, would you mind to change the key to getencoding?

```suggestion
    info_add('locale.getencoding', locale.getencoding())
```